### PR TITLE
Built header and footer for page layout

### PR DESCRIPTION
**Pull request by slroberts**
------------------
[Built header and footer](https://trello.com/c/0fNUiKH8)
-------------------------
### Location:
- RenderHomePage.js
### Details:
1) Built app's header and footer to define page layout
-------------------------
## Committed Features:
1 | Built out header and footer to define page layout.

### CODE REVIEW

**Review by BritneyParkerson — APPROVED** [review]
Great start for the layout!

File: src/components/pages/Home/RenderHomePage.js [hunk 1]
```diff
@@ -1,34 +1,38 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
-import { Button } from '../../common';
+import cityspireLogo from '../../../assets/imgs/cityspireLogo.png';
+
+import { Layout, Avatar, Button, Image, Space } from 'antd';
+
+import { UserOutlined } from '@ant-design/icons';
+
+const { Header, Footer } = Layout;
```
Comment (codyt11) [inline src/components/pages/Home/RenderHomePage.js:9]: thank you for spacing the imports it made it easier to read!

File: src/components/pages/Home/RenderHomePage.js [hunk 2]
```diff
@@ -1,34 +1,38 @@
...
+            {console.log(userInfo)}
+          </Space>
+          <Button onClick={() => authService.logout()}>Logout</Button>
+        </Space>
+      </Header>
+      <Footer style={{ textAlign: 'center' }}>
+        Cityspire ©2021 Created by Labspt15-cityspire-g
+      </Footer>
+    </Layout>
   );
```
Comment (codyt11) [inline src/components/pages/Home/RenderHomePage.js:36]: fantastic, well done, clean code!

**Review by codyt11 — APPROVED** [review]
Keep it up, I appreciate all the hard work!

**Pull request merged**